feature-libs/organization/administration/styles/_list.scss

```diff
@@ -261,6 +261,10 @@
         @include visible-focus();
         outline-offset: -2px;
       }
+
+      .invisible-focus-border {
+        outline-width: 0px;
```

Comment:
can we go with `outline: none`? or something as general and easy?

Reply:
outline:none is definitely more self-explained than outline-width: 0px. Thanks for this suggestion!